lib/index.js

```diff
@@ -0,0 +1,17 @@
+/**
```

Comment:
Hmm, I was under the impression this was the copy's standard copyright / description block.  I have no issue with leaving it in, or taking it out - just say the word :)